### ISSUE TITLE
Data analysis repo not found

### ISSUE THREAD

**Issue by agitter** [xomicsdatascience/transcriptome-proteome-nas-manubot#2]
I'm getting a 404 error for https://github.com/xomicsdatascience/RnaToProteinDataModule. Is it still a private repo?

**Comment by CCranney** [comment]
Yes it was, thank you for catching that! You should be able to access it now. I'm going to close this issue, feel free to open it up if you still have trouble accessing it.
